.github/workflows/tests.yml

```diff
@@ -24,7 +24,11 @@ jobs:
         #flake8 --ignore=E501,E123,E402 .
     - name: Test with pytest
       run: |
-        pytest tests
+        export test_level=2
+        export nval_lang=github
+        pytest tests/test_augments.py
```

Comment:
Can we change this back to `pytest tests` so that future additions of tests don't need to modify this file?

Reply:
Github actions was erroring out with exit code 137 when using `pytest tests` which I think is a result of `tests/test_augments.py` and `tests_test_bench.py` being in the same test session. That is, there was some overlap in having disk/memory from `tests/test_augments.py` when running `tests_test_bench.py` or vice versa. Implementing setup/teardown in `tests_test_bench.py` should resolve this so I'll do that before next PR.